beekeeper-cleanup/src/main/java/com/expediagroup/beekeeper/cleanup/aws/S3Client.java

```diff
@@ -73,17 +73,17 @@ void deleteObject(String bucket, String key) {
       return Collections.emptyList();
     }
     if (!dryRunEnabled) {
-      keys.forEach(key -> log.info("Deleting: \"{}/{}\"", bucket, key));
+      keys.forEach(key -> log.debug("Deleting: \"{}/{}\"", bucket, key));
```

Comment:
was this logged in info somewhere else?

Reply:
there's an info log for each path and then another one for each object under that path which i've now set to debug cause there are way too many and it's hard to find something in the logs

Reply:
So if we delete s3:bucket/table/partition-x what do we print? I just want to make sure that if someone complains that their path is deleted we can look it up. 
We can't switch to debug afterwards.
We can use Splunk to search in the logs so not sure what's hard to find.

Reply:
example log for 1 path: 
```
2021-11-10 17:06:37.650  INFO 1 --- [   scheduling-1] c.e.b.p.c.handler.GenericPathHandler     : Cleaning up path "s3://apiary-112945290135-us-east-1-user/clickstream_eg_domain_event_bot_free_v2/2021_10_17_191918/event_date=2021-10-16/event_hour=17"
2021-11-10 17:06:37.685  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/clickstream_eg_domain_event_bot_free_v2/2021_10_17_191918/event_date=2021-10-16/event_hour=17/000038_0"
2021-11-10 17:06:37.685  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/clickstream_eg_domain_event_bot_free_v2/2021_10_17_191918/event_date=2021-10-16/event_hour=17/000039_0"
2021-11-10 17:06:37.685  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/clickstream_eg_domain_event_bot_free_v2/2021_10_17_191918/event_date=2021-10-16/event_hour=17/000040_0"
2021-11-10 17:06:37.685  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/clickstream_eg_domain_event_bot_free_v2/2021_10_17_191918/event_date=2021-10-16/event_hour=17/000041_0"
2021-11-10 17:06:37.685  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/clickstream_eg_domain_event_bot_free_v2/2021_10_17_191918/event_date=2021-10-16/event_hour=17/000042_0"
2021-11-10 17:06:37.685  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/clickstream_eg_domain_event_bot_free_v2/2021_10_17_191918/event_date=2021-10-16/

Reply:
however, for the metadata we have:

``` 
2021-11-24 08:40:37.256  INFO 1 --- [   scheduling-1] c.e.b.m.c.h.ExpiredMetadataHandler       : Cleaning up metadata for "user.segments_batch_output".
2021-11-24 08:40:37.319  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.hive.HiveClient    : Dropping partition "segment_id=ab9117f8-4218-42ca-816a-4d82533dc795/run_dt=2021-11-10" from table "user.segments_batch_output"
2021-11-24 08:40:37.395  INFO 1 --- [   scheduling-1] c.e.b.c.m.DeletedMetadataReporter        : Deleted hive-partition
2021-11-24 08:40:37.456  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/mass/batch/output/segment_id=ab9117f8-4218-42ca-816a-4d82533dc795/run_dt=2021-11-10/_SUCCESS"
2021-11-24 08:40:37.456  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/mass/batch/output/segment_id=ab9117f8-4218-42ca-816a-4d82533dc795/run_dt=2021-11-10/part-00000-55c66a6d-9542-4474-972b-782c0d16863e-c000.snappy.parquet"
2021-11-24 08:40:37.456  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/mass/batch/output/segment_id=ab9117f8-4218-42ca-816a-4d82533dc795/run_dt=2021-11-10/part-00012-55c66a6d-9542-4474-972b-782c0d16863e-c000.snappy.parquet"
2021-11-24 08:40:37.456  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/mass/batch/output/segment_id=ab9117f8-4218-42ca-816a-4d82533dc795/run_dt=2021-11-10/part-00028-55c66a6d-9542-4474-972b-782c0d16863e-c000.snappy.parquet"
2021-11-24 08:40:37.456  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/mass/batch/output/segment_id=ab9117f8-4218-42ca-816a-4d82533dc795/run_dt=2021-11-10/part-00065-55c66a6d-9542-4474-972b-782c0d16863e-c000.snappy.parquet"
2021-11-24 08:40:37.456  INFO 1 --- [   scheduling-1] c.e.beekeeper.cleanup.aws.S3Client       : Deleting: "apiary-112945290135-us-east-1-user/mass/batch/output/segment_id=ab9117f8-4218-42ca-816a-4d82533dc795/run_dt=2021-11-10/
```

so if we turn off the s3Client logs then we'll have the name of the partition being deleted(`segment_id=ab9117f8-4218-42ca-816a-4d82533dc795/run_dt=2021-11-10`), but not the actual path(`apiary-112945290135-us-east-1-user/mass/batch/output/segment_id=ab9117f8-4218-42ca-816a-4d82533dc795/run_dt=2021-11-10`), not sure if that is good enough or if I should add another log for the path so that it's the same as the path cleaner in the comment above

Reply:
discussed, we'll leave all the logs as info and use splunk to search through them